packages/angular/src/generators/library/library.spec.ts

```diff
@@ -46,6 +46,19 @@ describe('lib', () => {
       tree = createTreeWithEmptyWorkspace(2);
     });
 
+    it('should run the library generator without erroring if the directory has a trailing slash', async () => {
+      // ACT
+      let error = null;
+      try {
+        await runLibraryGeneratorWithOpts({ directory: 'mylib/shared/' });
+      } catch (e) {
+        error = e;
+      } finally {
+        // ASSERT
+        expect(error).toBe(null);
+      }
```

Comment:
We could simplify this to:
```suggestion
      // ACT & ASSERT
      await expect(
        runLibraryGeneratorWithOpts({ directory: 'mylib/shared/' })
      ).resolves.not.toThrow();
```